### ISSUE TITLE
not an issue but a question

### ISSUE THREAD

**Issue by DeceptedPar** [realDunkin/FlashPad-Remake#1]
so... the megapack is cancelled huh?

**Comment by realDunkin** [comment]
this is like 3 years old at this point, but i should clarify what happened. 😭

Pretty much one day i somehow accidentally formatted my hard drive and lost everything on it, so i just gave up and forgot about the megapack. i wanted to get back into making it again but i can't get a hold of the source code. i even tried looking up if i could reverse engineer from the beta test html5 build but to no avail.

so thats what happened, dont know why i never said anything or updated you but yeah

**Comment by DeceptedPar** [comment]
damn, that's okay though bro, I mean, we did it for fun, I had never properly worked with someone to beta test and suggest stuff like that before, honestly nower days you have modpacks you can easily make or download so these types of things are basically rendered useless, though mic'd up never gets used which is kinda sad

thanks for updating me, even though this is two months after you sent the message

**Comment by realDunkin** [comment]
true, it was fun and my first time ever actually programming, even if most was copy and pasting but i won't downplay the work I did to make everything I added work side by side. I do like how you don't have to download the whole game or seperate mods, it can just be one thing. Recently I started work on a fnf client/mod that allows for mods to be loaded online through the game itself but its not supposed to be anything serious. if you are interested i am willing to actually work on this and make it optimized as I can, since in its current state it semi works, on windows it loads correctly but takes a while but html5 only goes up to the freeplay menu and cant load the actual images yet. rn it loads custom charts, songs, characters but not stages yet.
